packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -159,6 +159,12 @@ export interface AutoScalingGroupProps extends CommonAutoScalingGroupProps {
    * AMI to launch
    */
   machineImage: ec2.IMachineImageSource;
+
+  /**
+   * An IAM role to associate with the instance profile assigned to this Auto Scaling Group.
```

Comment:
Maybe mention that the role must be assumable by `{ Service: ec2.amazonaws.com }`

Reply:
Good call. Added